src/axom/primal/geometry/KnotVector.hpp

```diff
@@ -546,25 +546,25 @@ class KnotVector
   {
     SLIC_ASSERT(isValidSpan(span, t));
 
-    const int m_deg = getDegree();
+    const int deg = getDegree();
```

Comment:
It looks like all of the changes to this function can be undone if we remove this line since `getDegree()` just returns `m_deg`

@jcs15c -- could you please confirm?

Reply:
Yes, that's right, you can just remove the assignment. Sorry about that! Probably an artifact from when this was a function in `NURBSCurve` instead of in the dedicated `KnotVector` class.